code/Frontend/react-frontend/src/pages/LoginPage.js

```diff
@@ -50,7 +49,7 @@ function LoginPage() {
           const user = response.data;
           const username = user.username;
           const userType = user.user_type;
-          setUsername(username.split("@")[0]);
+          setUsername(username);
```

Comment:
Do we just use email as username instead of the first part of it going forward?

Reply:
Yes I think so -- that way it is more reliable. Since no one can have the same email but if they have the same username but on a different email account it might've gotten confused